tests/testthat/test-misc.R

```diff
@@ -160,7 +160,7 @@ test_that("default date formater", {
 
 test_that("Cubify works with many dimensions", {
     # 1d
-    cube_json <- fromJSON(test_path("cubes/univariate-categorical.json"))
+    cube_json <- fromJSON(system.file("cubes/univariate-categorical.json", package = "crunch"))
```

Comment:
I know you're not changing this here, but I don't understand these tests. AFAIK you can get `$result$measures$count$data`, unmodified from the server response, from the CrunchCube without having to `fromJSON` it separately.

Reply:
The original intent here was confirming the behavior of cubify with dimensions greater than 2. I wrote it this way to confirm that there was no re-arranging going on when we read cubes in. There isn't so we could change this to use the `$result$measures$count$data` if we wanted.